guides/doc-Release_Notes/topics/katello-4.19.0.adoc

```diff
@@ -0,0 +1,113 @@
+= Katello 4.19.0
```

Comment:
I'm not seeing these notes on https://theforeman-foreman-documentation-preview-pr-4466.surge.sh/3.17/Release_Notes/index-katello.html#katello-release-notes

I think a reference to this page needs to be added somewhere.